sdk/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -127,10 +127,26 @@ function Install-Tools(
 
     $DotNetHome = Resolve-Path $DotNetHome
     $installDir = if ($IS_WINDOWS) { Join-Path $DotNetHome 'x64' } else { $DotNetHome }
+    Write-Verbose "Installing tools to '$installDir'"
+    if ($env:DOTNET_INSTALL_DIR -and $env:DOTNET_INSTALL_DIR -ne $installDir) {
+        # DOTNET_INSTALL_DIR is used by dotnet-install.ps1 only, and some repos used it in their automation to isolate dotnet.
+        # DOTNET_HOME is used by the rest of our KoreBuild tools and is set by the bootstrappers.
+        Write-Verbose "installDir = $installDir"
+        Write-Verbose "DOTNET_INSTALL_DIR = ${env:DOTNET_INSTALL_DIR}"
+        Write-Warning 'It looks like a deprecated environment variable is set: $env:DOTNET_INSTALL_DIR. The recommended alternative is $env:DOTNET_HOME.'
```

Comment:
Maybe `The environment variable $env:DOTNET_INSTALL_DIR is deprecated...` just for simplicity.